src/components/Form/SelectDropdown.tsx

```diff
@@ -1,43 +1,57 @@
 import { useContext, useState } from "react";
 import { ColorSchemeContext } from "src/contexts/colorSchemeToggle/colorSchemeContext";
+import { StylesConfig } from "react-select";
+import { Option } from "src/types/index";
+import AsyncSelect from "react-select/async";
 import AsyncCreatableSelect from "react-select/async-creatable";
 import fuzzysort from "fuzzysort";
 
-import { Option } from "src/types/index";
-
 // Referencing tailwindcss styles in js would increase package size.
 // https://tailwindcss.com/docs/configuration#referencing-in-java-script
 // https://github.com/vercel/next.js/discussions/31841
 const colorsDark = {
   neutral0: "#171717", // neutral-900 background
-  neutral20: "#525252", // neutral-600 border
   neutral80: "#ffffff", // selected option text
   primary: "#3b82f6", // blue-500 focused
   primary25: "#1e40af", // blue-800 option hover
 };
+
 const colorsLight = {
-  neutral20: "#a3a3a3", // neutral-400 border
   primary: "#3b82f6", // blue-500 focused
   primary25: "#dbeafe", // blue-100 option hover
 };
 
-interface MultiSelectDropdownProps {
+const styleDark = {
+  border: 0,
+};
+
+const styleLight = {
+  borderColor: "rgba(0, 0, 0, 0.075)",
+};
+
+interface SelectDropdownProps {
   id: string;
   label: string;
   endpoint?: string;
   value: Option[];
   setValue: (value: Option[]) => void;
+  formType: AsyncSelect | AsyncCreatableSelect,
```

Comment:
Ya, I was thinking something similar, like making a constants file, but went this way cuz it was easier for now. Enums seems better though.

Reply:
I'm not sure I know what you mean by a constants file, but I feel like having the type of `formType` be `"normal" | "creatable"` would work well in this situation. Or, you could remove the `formType` prop and just have a `creatable` prop.

Reply:
So having a constants file defines constant variables that you can use anywhere. Instead of having to type out `"normal"` or `"creatable"` multiple times and potentially having to change all of them, you call `constants.NORMAL` or `constants.CREATABLE` only having to define the string value once.